src/main/kotlin/net/minecrell/pluginyml/GeneratePluginDescription.kt

```diff
@@ -45,14 +45,14 @@ import org.gradle.api.tasks.OutputDirectory
 import org.gradle.api.tasks.TaskAction
 import org.gradle.kotlin.dsl.property
 
-open class GeneratePluginDescription : DefaultTask() {
+abstract class GeneratePluginDescription : DefaultTask() {
 
     @Input
     val fileName: Property<String> = project.objects.property()
 
-    @Input
-    @Optional
-    val librariesConfiguration: Property<Configuration> = project.objects.property()
+    @get:Input
+    @get:Optional
+    abstract val librariesRootComponent: Property<ResolvedComponentResult>
```

Comment:
Is there a reason this Property is handled differently than the `fileName` now (abstract vs `project.objects.property()`?

Reply:
I would appreciate if you can make it consistent: Do you mind adding a second commit? (I guess strictly speaking it should come first, but on top would be also fine)